internal/network/utils.go

```diff
@@ -41,3 +45,21 @@ func GetClusterAddressStack(hosts []*models.Host) (bool, bool, error) {
 	}
 	return v4, v6, nil
 }
+
+func GetConfiguredAddressFamilies(cluster *common.Cluster) (ipv4 bool, ipv6 bool, err error) {
+	for _, cidr := range []string{cluster.MachineNetworkCidr, cluster.ClusterNetworkCidr, cluster.ServiceNetworkCidr} {
+		if cidr == "" {
+			continue
+		}
+		_, _, err = net.ParseCIDR(cidr)
+		if err != nil {
+			return false, false, errors.Wrapf(err, "%s id not a valid cidr", cidr)
```

Comment:
Looks like a type "% is not a valid CIDR" (written "id not")

Reply:
Correct